storybook/stories/contributing/contributing-documentation.txt

```diff
@@ -0,0 +1,77 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contributing/Documentation">
+
+This website is composed of articles with guidance on how we create products, which we call documentation.
+
+Our components and patterns need detailed documentation for usage, style, code, and accessibility — that way everyone can easily use them to do great work.
+
+We’d love for you to help us improve our documentation. It’s easy to get started.
+
+*Is this your first contribution? You’ll first need to follow <Link story="Contributing/Get Ready to Contribute">a couple of steps to get started</Link>.*
+
+### **Small copy changes**
+
+If you see a spelling, punctuation, or grammatical error anywhere in the documentation, you can help fix it. 
+
+You can suggest one or multiple small copy changes in a GitHub issue, which will then be approved and added by the community.
+
+[Click here to open a new issue](https://github.com/WordPress/gutenberg/issues/new?labels=Design+System&body=Then%2C+write+a+detailed+description%3A%0D%0A%0D%0A-+Explain+your+rationale+for+adding+this+component%0D%0A-+Detail+the+intended+behavior%0D%0A-+Clarify+if+it%E2%80%99s+a+new+component+of+a+variation+of+an+existing+one%0D%0A-+Include+a+basic+mockup+%28optional%29%0D%0A-+Include+inspiration+from+other+products+%28optional%29). Then, write a detailed description that explains the change(s) you’re making to the writing.
+
+After opening your issue, reviewers will start a discussion in the comments to see if it's an appropriate change. When it’s approved, the community will take care of making the change — unless you want to create a PR yourself.
+
+### **Adding new pages**
```

Comment:
I'm kind of doubting that a new contributor would be able to (or even want to) author a page from scratch on a foundational aspect of the design system, which requires a deep understanding of the whole.

I wonder if we should focus on improving flows for the more likely entry points for new contributors, which will be to edit existing pages. For example I could add some kind of footer on the doc pages, that links out to a GitHub view to edit the doc.